e2e/nomostest/clusters/kind.go

```diff
@@ -186,12 +186,13 @@ func createKindCluster(p *cluster.Provider, name, kcfgPath string, version KindV
 				},
 				// Enable ValidatingAdmissionWebhooks in the Kind cluster, as these
 				// are disabled by default.
+				// Also mount etcd to tmpfs for memory-backed storage.
 				KubeadmConfigPatches: []string{
 					`
-apiVersion: kubeadm.k8s.io/v1beta2
```

Comment:
Is apiVersion optional?

Reply:
apiVersion is set dynamically upstream by kind: https://github.com/kubernetes-sigs/kind/blob/v0.23.0/pkg/cluster/internal/kubeadm/config.go#L496